### PR TITLE
Doc 1.3 tweaks

### PR DESCRIPTION
**Pull request by carlescufi**
- Remove hal_nordic
- Include all boards

### CODE REVIEW

**Comment by carlescufi** [comment]
@jhn-nordic @pdunaj can you please verify that all boards are correct?

**Comment by carlescufi** [comment]
image

image

File: doc/nrf/gs_programming.rst [hunk 1]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+|                   |            |                                                                 | ``nrf9160dk_nrf9160ns``               |
+|                   |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf9160dk_nrf52840 <zephyr:nrf9160dk_nrf52840>`           | ``nrf9160dk_nrf52840``                |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+
+
+Boards included in sdk-nrf
+==========================
+
+Those boards are defined in ``nrf/boards/arm/<board name>``
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:226]:
```suggestion
The following boards are defined in the :file:`nrf/boards/arm/` folder.
```

File: doc/nrf/gs_programming.rst [hunk 2]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52840 DK       | PCA10056   | :ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`         | ``nrf52840dk_nrf52840``               |
+|                   |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf52840dk_nrf52811 <zephyr:nrf52840dk_nrf52811>`         | ``nrf52840dk_nrf52811``               |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52840 Dongle   | PCA10059   | :ref:`nrf52840dongle_nrf52840 <zephyr:nrf52840dongle_nrf52840>` | ``nrf52840dongle_nrf52840``           |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| Thingy:52         | PCA20020   | :ref:`thingy52_nrf52832 <zephyr:thingy52_nrf52832>`             | ``thingy52_nrf52832``                 |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:207]: Do we support Thingy:52?

File: doc/nrf/gs_programming.rst [hunk 3]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+Boards included in sdk-nrf
+==========================
+
+Those boards are defined in ``nrf/boards/arm/<board name>``
+
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
 | Hardware platform | PCA number | Board name                                               | Build target                          |
 +===================+============+==========================================================+=======================================+
-| nRF51 DK          | PCA10028   | :ref:`nrf51dk_nrf51422 <zephyr:nrf51dk_nrf51422>`        | ``nrf51dk_nrf51422``                  |
+| nRF Desktop       | PCA20041   | ``nrf52840gmouse_nrf52840``                              | ``nrf52840gmouse_nrf52840``           |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:231]:
```suggestion
| nRF Desktop       | PCA20041   | :ref:`nrf52840gmouse_nrf52840 <nrf_desktop>`                              | ``nrf52840gmouse_nrf52840``           |
```

File: doc/nrf/gs_programming.rst [hunk 4]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+| Gaming Mouse      |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52 DK          | PCA10040   | :ref:`nrf52dk_nrf52832 <zephyr:nrf52dk_nrf52832>`        | ``nrf52dk_nrf52832``                  |
+| nRF Desktop       | PCA20045   | ``nrf52810dmouse_nrf52810``                              | ``nrf52810dmouse_nrf52810``           |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:234]:
```suggestion
| nRF Desktop       | PCA20045   | :ref:`nrf52810dmouse_nrf52810 <nrf_desktop>`                              | ``nrf52810dmouse_nrf52810``           |
```

File: doc/nrf/gs_programming.rst [hunk 5]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+| Mouse             |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52833 DK       | PCA10100   | :ref:`nrf52833dk_nrf52833 <zephyr:nrf52833dk_nrf52833>`  | ``nrf52833dk_nrf52833``               |
+| nRF Desktop       | PCA20037   | ``nrf52kbd_nrf52832``                                    | ``nrf52kbd_nrf52832``                 |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:237]:
```suggestion
| nRF Desktop       | PCA20037   | :ref:`nrf52kbd_nrf52832 <nrf_desktop>`                                    | ``nrf52kbd_nrf52832``                 |
```

File: doc/nrf/gs_programming.rst [hunk 6]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+| Keyboard          |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52840 DK       | PCA10056   | :ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`  | ``nrf52840dk_nrf52840``               |
+| nRF Desktop       | PCA20044   | ``nrf52dmouse_nrf52832``                                 | ``nrf52dmouse_nrf52832``              |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:240]:
```suggestion
| nRF Desktop       | PCA20044   | :ref:`nrf52dmouse_nrf52832 <nrf_desktop>`                                 | ``nrf52dmouse_nrf52832``              |
```

File: doc/nrf/gs_programming.rst [hunk 7]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+| Mouse             |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF5340 PDK       | PCA10095   | :ref:`nrf5340pdk_nrf5340 <zephyr:nrf5340pdk_nrf5340>`    | ``nrf5340pdk_nrf5340_cpunet``         |
-|                   |            |                                                          |                                       |
-|                   |            |                                                          | ``nrf5340pdk_nrf5340_cpunetns``       |
-|                   |            |                                                          |                                       |
-|                   |            |                                                          | ``nrf5340pdk_nrf5340_cpuapp``         |
+| nRF Desktop       | PCA20111   | ``nrf52833dongle_nrf52833``                              | ``nrf52833dongle_nrf52833``           |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:243]:
```suggestion
| nRF Desktop       | PCA20111   | :ref:`nrf52833dongle_nrf52833 <nrf_desktop>`                              | ``nrf52833dongle_nrf52833``           |
```

File: doc/nrf/gs_programming.rst [hunk 8]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
+| Dongle            |            |                                                          |                                       |
++-------------------+------------+----------------------------------------------------------+---------------------------------------+
+| nRF Desktop       | PCA20114   | ``nrf52820dongle_nrf52820``                              | ``nrf52820dongle_nrf52820``           |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:246]:
```suggestion
| nRF Desktop       | PCA20114   | :ref:`nrf52820dongle_nrf52820 <nrf_desktop>`                              | ``nrf52820dongle_nrf52820``           |
```

File: doc/nrf/gs_programming.rst [hunk 9]
```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
...
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52840 Dongle   | PCA10059   | :ref:`nrf52840dongle_nrf52840 <zephyr:nrf52840dongle_nrf52840>` | ``nrf52840dongle_nrf52840``           |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| Thingy:52         | PCA20020   | :ref:`thingy52_nrf52832 <zephyr:thingy52_nrf52832>`             | ``thingy52_nrf52832``                 |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF5340 PDK       | PCA10095   | :ref:`nrf5340pdk_nrf5340 <zephyr:nrf5340pdk_nrf5340>`           | ``nrf5340pdk_nrf5340_cpunet``         |
+|                   |            |                                                                 |                                       |
+|                   |            |                                                                 | ``nrf5340pdk_nrf5340_cpunetns``       |
```
Comment (anangl) [inline doc/nrf/gs_programming.rst:211]: There is no such target. It should be `nrf5340pdk_nrf5340_cpuappns`.

File: doc/nrf/gs_programming.rst (hunk 2 above)
Comment (carlescufi) [reply doc/nrf/gs_programming.rst:207]: Well, Zephyr does

File: doc/nrf/gs_programming.rst (hunk 9 above)
Comment (carlescufi) [reply doc/nrf/gs_programming.rst]: Fixed, thank you

File: doc/nrf/gs_programming.rst [hunk 10]
```diff
@@ -178,31 +178,78 @@ For more information on building and programming using the command line, see the
...
+| Mouse             |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF5340 PDK       | PCA10095   | :ref:`nrf5340pdk_nrf5340 <zephyr:nrf5340pdk_nrf5340>`    | ``nrf5340pdk_nrf5340_cpunet``         |
-|                   |            |                                                          |                                       |
-|                   |            |                                                          | ``nrf5340pdk_nrf5340_cpunetns``       |
-|                   |            |                                                          |                                       |
-|                   |            |                                                          | ``nrf5340pdk_nrf5340_cpuapp``         |
+| nRF Desktop       | PCA20111   | :ref:`nrf52833dongle_nrf52833 <nrf_desktop>`             | ``nrf52833dongle_nrf52833``           |
```
Comment (pdunaj) [inline doc/nrf/gs_programming.rst:244]: pca10111

File: doc/nrf/gs_programming.rst [hunk 11]
```diff
@@ -178,31 +178,78 @@ For more information on building and programming using the command line, see the
...
+| Dongle            |            |                                                          |                                       |
++-------------------+------------+----------------------------------------------------------+---------------------------------------+
+| nRF Desktop       | PCA20114   | :ref:`nrf52820dongle_nrf52820 <nrf_desktop>`             | ``nrf52820dongle_nrf52820``           |
```
Comment (pdunaj) [inline doc/nrf/gs_programming.rst:247]: pca10114

File: doc/nrf/gs_programming.rst (hunk 2 above)
Comment (ru-fu) [reply doc/nrf/gs_programming.rst:208]: Zephyr also supports nRF51 DK.

File: doc/nrf/gs_programming.rst [hunk 12]
```diff
@@ -178,31 +178,78 @@ For more information on building and programming using the command line, see the
...
+| Mouse             |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52833 DK       | PCA10100   | :ref:`nrf52833dk_nrf52833 <zephyr:nrf52833dk_nrf52833>`  | ``nrf52833dk_nrf52833``               |
+| nRF Desktop       | PCA20037   | :ref:`nrf52kbd_nrf52832 <nrf_desktop>`                   | ``nrf52kbd_nrf52832``                 |
+| Keyboard          |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52840 DK       | PCA10056   | :ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`  | ``nrf52840dk_nrf52840``               |
+| nRF Desktop       | PCA20044   | :ref:`nrf52dmouse_nrf52832 <nrf_desktop>`                | ``nrf52dmouse_nrf52832``              |
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst:241]: Is there a reason for this order? I would list all mouse devices first, then the keyboard.

**Comment by carlescufi** [comment]
@pdunaj @ru-fu thanks for the review. Comments addressed.

**Pull request merged**